.circleci/config.yml

```diff
@@ -377,7 +377,7 @@ jobs:
           name: Test
           command: |
             cd code
-            yarn test --coverage --ci --runInBand
```

Comment:
@yannbf and I found that this flag is required in order for the tests to pass in circleCI.  Otherwise, it stalls out after 10 minutes.  With the flag, it completes very quickly.